dataactbroker/permissions.py

```diff
@@ -38,12 +42,20 @@ def decorated_function(*args, **kwargs):
                 elif LoginSession.isLogin(session):
                     user = sess.query(User).filter(User.user_id == session["name"]).one()
                     valid_user = True
-                    if permission in PERMISSION_TYPE_DICT and \
-                            not user.permission_type_id == PERMISSION_TYPE_DICT[permission]:
-                        valid_user = False
+
+                    if permission is not None and not user.permission_type_id == PERMISSION_TYPE_DICT['website_admin']:
```

Comment:
Isn't `website_admin` the top permission? If so, why does it need to be excluded?